operator/docs/user-guides/open-telemetry.md

```diff
@@ -0,0 +1,163 @@
+
+```yaml
+# [...]
+spec:
+  limits:
+    global:
+      otlp: {} # Global OTLP Attribute Configuration
+    tenants:
+      example-tenant:
+        otlp: {} # OTLP Attribute Configuration for tenant "example-tenant"
+```
+
+Both global and per-tenant OTLP configuration can contain rules mapping attributes to stream-labels or structured-metadata. At least _one stream-label_ is needed for successfully saving a log entry to Loki storage, so the configuration should account for that.
+
+Stream labels can only be generated from resource-level attributes, which is mirrored in the data structure of the `LokiStack` resource:
+
+```yaml
+# [...]
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          resourceAttributes:
+          - name: "k8s.namespace.name"
+          - name: "k8s.pod.name"
+          - name: "k8s.container.name"
+```
+
+Structured metadata on the other hand can be generated from all types of attributes (resource, scope and log):
+
+```yaml
+# [...]
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          # [...]
+        structuredMetadata:
+          resourceAttributes:
+          - name: "process.command_line"
+          - name: "k8s\\.pod\\.labels\\..+"
+            regex: true
+          scopeAttributes:
+          - name: "service.name"
+          logAttributes:
+          - name: "http.route"
+```
+
+The previous example also shows that the attribute names can be expressed as _regular expressions_ by setting `regex: true`.
+
+Using a regular expression makes sense when there are many attributes with similar names that should be mapped into Loki. It is not recommended to be used for stream labels, as it can potentially create a lot of data.
+
+### Customizing OpenShift Defaults
+
+The `openshift-logging` tenancy mode contains its own set of default attributes. Some of these attributes (called "required attributes") can not be removed by applying a custom configuration, because they are needed for other OpenShift components to function properly. Other attributes (called "recommended attributes") are provided but can be disabled in case they influence performance negatively. The complete set of attributes is documented in the [data model](rhobs-data-model) repository.
+
+Because the OpenShift attribute configuration is applied based on the tenancy mode, the simplest configuration is to just set the tenancy mode and not apply any custom attributes. This will provide instant compatibility with the other OpenShift tools.
+
+In case additional attributes are needed, either as stream labels or structured metadata, the normal custom attribute configuration mentioned above can be used. In contrast to the Grafana default labels, providing a custom attribute configuration in the `openshift-logging` tenancy mode does not remove the defaults. Instead the custom configuration is **merged** with the default configuration.
```

Comment:
Since the first paragraph already mentions that some defaults cannot be disabled what do you think of omitting that part from this section?
```suggestion
In case additional attributes are needed, either as stream labels or structured metadata, the normal custom attribute configuration mentioned above can be used. Attributes defined in the custom configuration will be **merged** with the default configuration.
```